Gemfile.lock

```diff
@@ -41,13 +41,18 @@ GEM
       execjs
     bcrypt (3.1.11)
     bcrypt (3.1.11-x86-mingw32)
+    better_errors (2.1.1)
```

Comment:
The file gives an error after indentation, had to do bundle install and positioned back